connectomics/data/utils/data_io.py

```diff
@@ -23,6 +23,11 @@ def readvol(filename, dataset=''):
         raise ValueError('unrecognizable file format for %s'%(filename))
     return data
 
+def savevol(filename, vol, dataset='main'):
```

Comment:
This is the savevol function, which is very similar to writeh5. Is this implementation fine?